scripts/installer/install.sh

```diff
@@ -405,6 +406,7 @@ else
     echo "Error: Invalid root fs" >&2
     exit 1
 fi
+echo "Finished writing root filesystem."
```

Comment:
Is it "filesystem" or "file system"? Please decide on one.

Reply:
This is getting silly. I _literally_ took a suggestion which I didn't think was needed in the first place.

Changed.